chapters/CRUD-Operations.md

```diff
@@ -5,83 +5,200 @@ for the **did:btcr2** method.
 
 ### Create
 
-The Create operation consists of two main algorithms for creating identifiers
-and DID documents. A **did:btcr2** identifier and DID document can either be created
-from a deterministic key pair or from an external ::Intermediate DID Document::.
-In both cases, DID creation can be undertaken in an offline manner, i.e., the DID
-controller does not need to interact with the Bitcoin network to create their DID.
+Creating a **did:btcr2** identifier is entirely offline, requiring no innate
```

Comment:
We might want to put here. Creating an identifier does not require onchain anchoring following Ryans language suggestion